Game/RoomsAndDoors/Rooms.cs

```diff
@@ -94,18 +98,56 @@ public void Draw()
             }
         }
 
-        public Room getNextRoom()
+        public Room getNorthRoom()
+        {
+            if (changeTimeDelay < 0 && connectingRooms[0] != null)
```

Comment:
For using keys to change room checking if the room exists is fine, but besides the starting room it would be impossible to leave a door and not have a room behind it. Since the starting room is differentiated we can manually make sure the player doesn't try to load a room to the south. It's fine to keep it for now though to make sure during debugging we don't mistakenly load a room that doesn't exist.